packages/cli/src/lib/defaults/language-overrides/app/python/index.ts

```diff
@@ -0,0 +1,31 @@
+import { CodegenOverrides } from "../../../../codegen";
+import { PolywrapProject } from "../../../../project";
+import { intlMsg } from "../../../../intl";
+
+import fs from "fs";
+import path from "path";
+
+export function getCodegenOverrides(): CodegenOverrides {
+  return {
+    getSchemaBindConfig: async (project: PolywrapProject) => {
+      const manifestPath = project.getManifestPath();
+      const manifestDir = path.dirname(manifestPath);
+      const pyprojectPath = path.join(manifestDir, "pyproject.toml");
```

Comment:
Isn't it also possible for python projects to not use a `pyproject.toml` config file, for example if they use `requirements.txt`?

In the case we don't find a `pyproject.toml` file, can we just default to `./wrap` and not throw?

Reply:
https://peps.python.org/pep-0621/
[PEP 621 – Storing project metadata in pyproject.toml | peps.python....](https://peps.python.org/pep-0621/)
Pyproject.toml is the standard way to declare project metadata for Python projects.